src/pages/core/standard-library/cryptography.mdx

```diff
@@ -20,8 +20,7 @@ documentation about them.
 
 <Callout>
   Note that these values are in CosmWasm Gas (which is very much different from
-  Cosmos SDK Gas). For more info on gas, [check this
-  document](https://github.com/CosmWasm/cosmwasm/blob/main/docs/GAS.md).
+  Cosmos SDK Gas). For more info on gas, [check this document].
```

Comment:
Same for this. It got highlighted because of formatting changes. But we can move those two points you raised into own issues and handle them separately.